aws-ssm-patchbaseline/inputs/inputs_1_read.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
is this used in the test?

Reply:
Actually i have used this this files for CFN test case.. The test case process also not using this json files for testing payload. Now i have removed the files